h/session.py

```diff
@@ -18,6 +18,15 @@ def pop_flash(request):
             for k in ['error', 'info', 'warning', 'success']}
 
 
+def _group_sort_key(group):
+    """Sort private groups for the session model list"""
+
+    # groups are sorted first by name but also by ID
+    # so that multiple groups with the same name are displayed
+    # in a consistent order in clients
+    return '%s:%s' % (group.name.lower(), group.hashid)
```

Comment:
It would probably be more pythonic (and not dependent on the sorting properties of `':'`) to return a tuple here: `return (group.name.lower(), group.hashid)`